ubcpi/static/html/ubcpi_edit.html

```diff
@@ -0,0 +1,153 @@
+
+                <li class="field comp-setting-entry is-set">
+                    <div class="wrapper-comp-setting">
+                        <label class="label setting-label pi-setting-label" for="pi-option-correct">Correct Option<span class="tip setting-help">This is the correct option. This enables you to know if a student answered the question correctly.</span></label>
+                        <select name="pi-option-correct" id="pi-option-correct" ng-model="esc.data.correct_answer" ng-model-options="{ debounce: 500 }" ng-options="esc.data.options.indexOf(option) as ('Option ' + (esc.data.options.indexOf(option) +1) ) for option in esc.data.options" required></select>
+                    </div>
+                    <div class="wrapper-comp-setting">
+                        <label class="label setting-label pi-setting-label" for="pi-correct-rationale">Rationale<span class="tip setting-help">Explain to students why this is the correct option.</span></label>
+                        <textarea name="pi-correct-rationale" id="pi-correct-rationale" ng-model="esc.data.correct_rationale.text" ng-model-options="{ debounce: 500 }" required></textarea>
+                    </div>
+                </li>
+
+                <li class="field comp-setting-entry is-set">
+                    <div class="wrapper-comp-setting">
+                        <label class="label setting-label pi-setting-label" for="pi-algo">Selection Algorithm<span class="tip setting-help">The algorithm for determining which other answers the student answering this question sees after they submit their answer and rationale.</span></label>
+                        <select name="pi-algo" id="pi-algo" ng-model="esc.data.algo.name" ng-options="algo as algo for (algo, desc) in esc.algos" required></select>
+                    </div>
+                    <span class="tip setting-help">{{ esc.algos[esc.data.algo] }}</span>
+                    <div class="wrapper-comp-setting">
+                        <label class="label setting-label pi-setting-label" for="pi-algo-num-responses">Number of Responses<span class="tip setting-help">The number of responses presented to the students. Enter # for the same number as options.</span></label>
+                        <input class="input setting-input pi-options" name="pi-algo-num-responses" id="pi-algo-num-responses" ng-model="esc.data.algo.num_responses" ng-model-options="{ debounce: 500 }" type="text" required />
+                    </div>
+                </li>
+
+                <li class="field comp-setting-entry is-set">
+
+                    <label class="label setting-label" for="">Seeded Answers<span class="tip setting-help">Provide answers which can be shown to the students answering this question when there are not enough submitted responses from which to choose. </span><input type="button" class="button action-primary add-image-button" value="Add Seed" ng-click="esc.addSeed()" name="pi-option-add-seed" /></label>
```

Comment:
3rd thing too, I realize I misunderstood the UI here. If the button text was "Add New Seed" or "Add Another Seed", it would be clearer to understand you don't have to click that button to save the seed's value.

I still think it should be in the bottom right, so if you're adding 10 seeds you don't have to scroll up and down a bunch.